dsaps/models.py

```diff
@@ -51,26 +53,37 @@ def get_record(self, uuid, rec_type):
 
     def filtered_item_search(self, key, string, query_type,
                              selected_collections=''):
+        """Search using filtered item endpoint."""
         offset = 0
         items = ''
         item_links = []
         while items != []:
-            endpoint = f'{self.url}/rest/filtered-items?'
+            endpoint = f'{self.url}filtered-items?'
             params = {'query_field[]': key, 'query_op[]': query_type,
                       'query_val[]': string, '&collSel[]':
                       selected_collections, 'limit': 200, 'offset': offset}
-            logger.info(params)
-            print(endpoint)
             response = requests.get(endpoint, headers=self.header,
                                     params=params, cookies=self.cookies)
-            print(f'Response url: {response.url}')
+            logger.info(f'Response url: {response.url}')
             response = response.json()
             items = response['items']
             for item in items:
                 item_links.append(item['link'])
             offset = offset + 200
         return item_links
 
+    def post_coll_to_comm(self, base_url, comm_handle, coll_name):
+        """Post collection to specified community."""
+        endpoint = f'{self.url}handle/{comm_handle}'
+        community = requests.get(endpoint, headers=self.header,
+                                 cookies=self.cookies).json()
```

Comment:
This should probably have been caught in a previous PR, but `self.header` and `self.cookies` should be initialized in the `__init__` method. They can be initialized to `None` or whatever makes sense, but as it stands, if the `authenticate()` method has not been called this will raise an `AttributeError`.